op-node/rollup/status/status_test.go

```diff
@@ -0,0 +1,49 @@
+package status
+
+import (
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/engine"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/stretchr/testify/require"
+)
+
+type NoopMetrics struct{}
+
+func (m NoopMetrics) RecordL1ReorgDepth(d uint64)                 {}
+func (m NoopMetrics) RecordL1Ref(name string, ref eth.L1BlockRef) {}
+
+func TestStatus(t *testing.T) {
+
+	tracker := NewStatusTracker(testlog.Logger(t, log.LevelDebug), NoopMetrics{})
+
+	status := tracker.SyncStatus()
+	require.Equal(t, eth.SyncStatus{}, *status)
+
+	oneHundred := eth.L2BlockRef{Number: 100}
+	tracker.OnEvent(engine.ForkchoiceUpdateEvent{
+		UnsafeL2Head:    oneHundred,
+		SafeL2Head:      oneHundred,
+		FinalizedL2Head: oneHundred,
```

Comment:
better use three distinct numbers in this test so we can assert that the right fields are set.